.github/workflows/ci_windows.yaml

```diff
@@ -121,7 +121,7 @@ jobs:
             ${{ env.libtorrent_path }}
           cd ${{ env.libtorrent_path }}
           $env:CXXFLAGS+=" /guard:cf"
```

Comment:
How about this one?

Reply:
This one is for compiler, the other one is for linker. Different.

Reply:
> This one is for compiler, the other one is for linker.

I see it myself. Do they intentionally have different capitalization?

Reply:
> > This one is for compiler, the other one is for linker.
> 
> I see it myself. Do they intentionally have different capitalization?

It looks like it is.